### ISSUE TITLE
[RFC] Assignment Submission policy

### ISSUE THREAD

**Issue by sakethramanujam** [GVP-AI-Club/Organization-Guidelines#2]
:wave: Hello folks long since we've had this mechanism of sending out assignment problems to the peers that we on-board we've not decided on a flow/policy on how to evaluate the assignments/solutions.
This issue is open to all such kinds of discussions!

**Comment by sakethramanujam** [comment]
When we first started this org, @rakhi2104  and I decided that we'd have a common repo for all the members that join in. This structure was inspired by the flow on [Campus Experts](https://githubcampus.expert) training which was then the version 2.0 and is now considered a legacy system there.
This is to give you a brief idea of what we decided on, 
We have a base repository `Member-Submissions` which would contain one folder each for user and every user would have their solutions pushed/filed a pr

```
`Member-Submissions/`

------------------------------ Saketh Ramanujam/
---------------------------------------------------------/a.py
---------------------------------------------------------/b.py
------------------------------ Rakesh/
---------------------------------------------------------/a.py
---------------------------------------------------------/b.py
---
...
...
------------------------------Vinay/
```

The problems started when everybody started creating new subfolders on local without a ref to the remote origins and the remote i.e., the repo got very messy that @V1NAY8 spent some very productive time re-basing and merging everything and ultimately came up with an idea where he created one branch per individual with their files in it. The structure looked like

```
----------------------------------master------------------------------------------
    \        \        \         \     ................................................................\
   SR     RP    VP     SK                                                                   NK
```
That seem's to have worked for sometime.

**Comment by sakethramanujam** [comment]
Ever since the [GitHub Classroom](classroom.github.com) tool has been made availableI(also thanks to our friends at github who were gracious enough to give us access to tools that aid adding people to our repo for free as a part of the education program ) the world outside has been able to deal peacefully on managing assignments. FunFact it is also how the `campus-experts` training works now.!

Long story short, I think that it's time that we setup assignments using github classroom and add starter code with some testing scripts which would automate the process of evaluation and scoring.
